HW10/printff.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+FOLD=$2
+FTYPE1="*.png"
+FTYPE2="*.sh"
+
+function findFiles {
```

Comment:
Try to do your function more flexible - pass parameter to your function.
Hint: https://stackoverflow.com/questions/6212219/passing-parameters-to-a-bash-function